transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py

```diff
@@ -169,6 +171,26 @@ def df_to_services(revision: DatasetRevision, df: pd.DataFrame) -> Iterator[Serv
         )
 
 
+def df_to_vehicle_journeys(df: pd.DataFrame) -> Iterator[VehicleJourney]:
+    for record in df.to_dict("records"):
+        yield VehicleJourney(
+            journey_code=record["journey_code"],
+            start_time=record["departure_time"],
+            line_ref=record["line_ref"],
+            direction=record["direction"],
+        )
+
+
+def df_to_serviced_organisations(df: pd.DataFrame) -> Iterator[ServicedOrganisations]:
+    seen_names = set()
+    for record in df.to_dict("records"):
+        name = record["name"]
+
+        if name not in seen_names:
+            seen_names.add(name)
+            yield ServicedOrganisations(name=name)
```

Comment:
We could pass the `existing_serviced_orgs` and get a list of unique names. This list can be iterated to yield `ServicedOrganisations `

```df.loc[~df['name'].isin(existing_serviced_orgs), 'name'].unique()```

Reply:
Yeah, this prevents list comprehension outside this function